ksql-parser/src/main/java/io/confluent/ksql/parser/DefaultKsqlParser.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright 2018 Confluent Inc.
```

Comment:
nit: 2019

Reply:
Nah, Github may think this is a new file, but the code in this file comes from `KsqlParser.java` so I've maintained the original copyright date.